app/utils/metadata.py

```diff
@@ -44,7 +38,7 @@
 
 
 class MetadataManager(QObject):
-    _instance: Optional["MetadataManager"] = None
+    _instance = None  # type: MetadataManager
     mod_created_signal = Signal(str)
```

Comment:
Pylance dislikes this. Try the following:

```suggestion
    _instance: "None | MetadataManager" = None
```